backend/danswer/configs/app_configs.py

```diff
@@ -396,6 +396,10 @@
 )
 
 
+VESPA_REQUEST_TIMEOUT = int(os.environ.get("VESPA_REQUEST_TIMEOUT") or "0") or None
```

Comment:
Who is complaining about this? I mean this is fine to do but I am concerned that this is masking other issues

Reply:
Would this also correctly be a no-op in the case where it's not set? I think the default is like 5 seconds or something right? Would this be correct if it were None?